MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> We can't use OpenTK to make SDL window resizable.

You have completely misunderstood me.

Obviously SDL can't be fixed by OpenTK...   I was just trying to figure out if the problem existed in OpenTK (with native backend).

> PS. SDL 1.2 did support changing resizable parameter after window creation.

So that settles it...  SDL2 lost this feature and it is totally possible to change the resize window state on Mac and Linux platforms after window creation.

This was all I was trying to figure out.